Dom-mainpulation.js

```diff
@@ -1,14 +1,481 @@
+          ${arr.technologiesD[2]}
+          </li>
+        </ul>
+        <ul>
+          <li>
+          ${arr.technologiesD[3]}
+          </li>
+          <li>
+          ${arr.technologiesD[4]}
+          </li>
+          <li>
+          ${arr.technologiesD[5]}
+          </li>
+        </ul>
+      </div>
+      <nav class="see2">
+        <a href="${arr.linklive}" target="_blank" class="regular-flex2">
+        See live <span> <img src="/seelive.png" alt="live"> </span> 
+        </a>
+        <a href="${arr.linksource}" target="_blank" class="regular-flex2">
+          See source <span> <img src="/littlegit.png" alt="git"> </span> 
+          </a>
+      </nav>
+    </div>
+</div>
+</div>`;
+
+  Popdiv.insertAdjacentHTML('beforeend', divContent);
+  // Close function
+  const closebtn = document.querySelector('.Modalb');
+  closebtn.classList.add('close-x');
+  document.querySelector('.close-x').addEventListener('click', () => {
+    const window = document.querySelector('.pop-up');
+    window.classList.replace('active', 'not-active');
+    window.remove();
+  });
+}
+
+document.getElementById('see-project1').addEventListener('click', addModal1);
+
+// Second card
+function addModal2() {
+  const Popdiv = document.createElement('div');
+  Popdiv.classList.add('active');
+  bodyDiv.appendChild(Popdiv);
+  const arr = {
+    name: 'Multi-Post Stories', description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent", descriptionD: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea", image: '/multiphoto.png', imageD: '/2desktop.png', technologies: ['html', 'css', 'javascript'], technologiesD: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstraps'], linklive: 'https://yayoamigo.github.io/mobile-template/', linksource: 'https://github.com/yayoamigo/mobile-template',
+  };
```

Comment:
- [ ] I must commend your approach to displaying the project data dynamically 👍🏽 . I think it will be better if you add all your project data in a single array so that it will be easier to loop through the project data instead of repeating yourself 😄 

e.g

```
const projectDatas = [
   {
  name: 'Multi-Post Stories',
  workName: 'FACEBOOK',
  position: ['Full Stack Dev', '2015'],
  imageUrl: './img/Snapshoot Portfolio2.svg',
  description:
    'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and',
  technologies: ['html', 'css', 'javascript'],
},
{
  name: 'Multi-Post Stories',
  workName: 'FACEBOOK',
  position: ['Full Stack Dev', '2015'],
  imageUrl: './img/Snapshoot Portfolio2.svg',
  description:
    'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and',
  technologies: ['html', 'css', 'javascript'],
},
{
  name: 'Multi-Post Stories',
  workName: 'FACEBOOK',
  position: ['Full Stack Dev', '2015'],
  imageUrl: './img/Snapshoot Portfolio2.svg',
  description:
    'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and',
  technologies: ['html', 'css', 'javascript'],
};
]
```